2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/2.umap_on_selected_features.ipynb

```diff
@@ -34,6 +34,7 @@
     "import pandas as pd\n",
```

Comment:
<p>You can definitely see the clustering here</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>